tests/app/authentication/test_authentication.py

```diff
@@ -33,7 +38,12 @@ def test_should_not_allow_request_with_incorrect_header(client, auth_fn):
     request.headers = {"Authorization": "Basic 1234"}
     with pytest.raises(AuthError) as exc:
         auth_fn()
-    assert exc.value.short_message == "Unauthorized, authentication bearer scheme must be used"
+    assert (
+        exc.value.short_message
+        == "Unauthorized, Authorization header is invalid. "
+        + "Notify supports the following authentication methods. "
+        + ", ".join([f"{auth_type[0]}: {auth_type[2]}" for auth_type in AUTH_TYPES])
```

Comment:
Can you not use the constant here to make sure that the code we're testing lists all auth types properly? This bit of the test repeats the actual logic otherwise.